azerothcore/azerothcore-wotlk
# fix(DB/Event): Shadowfang Keep's Valentine decoration.

Improve on #12404 (and finish it, unless I've missed something)
## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13341
- Closes https://github.com/chromiecraft/chromiecraft/issues/4164
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12254 (forgotten + was partial)

## SOURCE:

https://youtu.be/PQOifDDT6z8?t=1160
## Tests Performed:

- tested


## How to Test the Changes:


.go c 16254
confirm that flowers and pink candles on table are gone, as well as the wall ornaments
.go c id 4274 (only wall ornaments)

.event start 8
Confirm that they are there when the event is up. (reset instance)

<details>
<summary> Flower vase + pink candle </summary>

![a8e6e2d4f1478b83f6a880ff99200576](https://user-images.githubusercontent.com/46330494/222977869-020e1605-a4f5-497d-a508-a871de84d9cf.png)

</details>


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
